### PR TITLE
refactor(logger): Add dataFormat to LogEntry

### PR DESCRIPTION
**Pull request by 10ko**
**What this PR does / why we need it**:
This PR introduces the ability of passying a `dataFormat` to the logging methods like:
```js
const someData: {
  key: "value",
}
log.info({ data: someData, dataFormat: "json" })
```
This way we can choose if we want to render the data as `json` or `yaml` (the current default).

The second commit removes some js files I found while writing the tests and adds a new tslint rule [tslint-no-focused-test](https://github.com/jasonmendes/tslint-no-focused-test) which fails if you forget `only` in your tests. :)

### CODE REVIEW

File: garden-service/test/unit/src/logger/log-entry.ts [hunk 1]
```diff
@@ -66,6 +66,7 @@ describe("LogEntry", () => {
       symbol: undefined,
       status: undefined,
       data: undefined,
+      dataFormat: undefined,
```
Comment (eysi09) [inline garden-service/test/unit/src/logger/log-entry.ts]: Could you add a value for `dataFormat` in the `"should update entry state"` test to check that it gets set. That is, set `dataFormat: "json"` in the `setState` call and check that the expected object also has `dataFormat: "json"`.

Comment (10ko) [reply garden-service/test/unit/src/logger/log-entry.ts]: Makes sense. Done.

**Comment by 10ko** [comment]
Can we merge this for 0.11.1?

**Comment by 10ko** [comment]
Can we merge this for 0.11.1?

**Pull request merged**